readme.md

```diff
@@ -180,14 +179,14 @@ Add the following extensions to your TwigBridge config/extensions.php (or regist
 The Dump extension will replace the [dump function](http://twig.sensiolabs.org/doc/functions/dump.html) to output variables using the DataFormatter. The Debug extension adds a `debug()` function which passes variables to the Message Collector,
 instead of showing it directly in the template. It dumps the arguments, or when empty; all context variables.
 
-```
+```twig
```

Comment:
Shouldn't this be `blade` instead of `twig`? Ref: https://laravel-news.com/2016/07/github-now-supports-blade-syntax-highlighting/